test/test_geant4.jl

```diff
@@ -66,6 +67,11 @@ end
     @test evts isa Table
     @test length(evts) == 100
 
+    # Cluster events by radius and time
+    @test_nowarn clusted_evts = SolidStateDetectors.cluster_detector_hits(evts, 10u"µm", 100u"s")
```

Comment:
I'm sorry, my previous message had a typo which is causing these tests to fail :sweat_smile: 
```suggestion
    @test_nowarn clustered_evts = SolidStateDetectors.cluster_detector_hits(evts, 10u"µm", 100u"s")
```